src/login/components/LoginApp/ResetPassword/SecurityKey.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import InjectIntl  from "../../../translation/InjectIntl_HOC_factory";
+import ResetPasswordLayout from "./ResetPasswordLayout";
+import PropTypes from "prop-types";
+
+const SecurityKey = (props) => {
+  return (
+    <ResetPasswordLayout
+      heading={props.translate("resetpw.extra-security_heading")} 
+      description={props.translate("resetpw.extra-security_description")} 
+      linkInfoText={props.translate("resetpw.without_extra_security")}
+      linkText={props.translate("resetpw.continue_reset_password")}
+    > 
+      <p>{props.translate("mfa.reset-password-tapit")}</p>
+      <div className="key-animation"  />
+      <div>
+        <form method="POST" action="#" id="form" className="form-inline">
```

Comment:
There is a component in the login flow called `SubmitSamlResponse` that might be useful to you here.

Reply:
ok, thanks
I will have a look